app/models/configuration_set.rb

```diff
@@ -31,6 +31,22 @@ def self.one_column(options={})
     configuration_set
   end
 
+  def self.two_column(options={})
+    default_template_path = '_cascade/templates/modular/two_column.html'
+    configuration_set = ConfigurationSet.new(name: '2 Column')
+    configuration_set.set_defaults
+    configuration_set.template = options.fetch(:template, default_template_path)
+    configuration_set
+  end
+
+  def self.three_column(options={})
+    default_template_path = '_cascade/templates/modular/three_column.html'
+    configuration_set = ConfigurationSet.new(name: '3 Column')
+    configuration_set.set_defaults
+    configuration_set.template = options.fetch(:template, default_template_path)
+    configuration_set
+  end
+
```

Comment:
Similar to my last two comments about refactoring those `self.default` class methods. I just personally like avoiding using a whole line to return an object like here with `configuration_set`.  I feel like it looks a little messy and it doesn't strike me as Ruby best practice if it's avoidable. I really try to take advantage of the Ruby behavior of returning whatever is produced by the last line of the method.

Not sure if there's such a simple solution here as there was on the last two methods just from precursory glance, but that's my 2 cents on Ruby philosophy.